app/views/home.njk

```diff
@@ -38,7 +38,7 @@
     {% if auth.isLedgerUser %}
     <div class="govuk-cards govuk-cards--2">
         <div class="govuk-cards--2__item govuk-card--dashboard">
-          <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Awaiting Manual Ledger Assignment</h2>
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Awaiting ledger assignment</h2>
```

Comment:
Final comment from my perspective:

The link under this reads `View Manual Ledger Assignment` should this be `View awaiting ledger assignment` to be consistent with the title and the other links on the page?

Reply:
agree with this. updated as suggested